composeApp/src/commonMain/kotlin/sp/bvantur/tasky/event/presentation/CreateEventViewModel.kt

```diff
@@ -226,6 +252,63 @@ class CreateEventViewModel(dispatcherProvider: DispatcherProvider, savedStateHan
         }
     }
 
+    private fun onAttendeeEmailChange(text: String) {
+        if (viewStateFlow.value.isAttendeeEmailError) {
+            emitViewState { viewState ->
+                viewState.copy(
+                    isAttendeeEmailError = validateEmailUseCase(text)
+                )
+            }
+        }
+    }
+
+    private fun onInviteAttendee(email: String) {
+        val isEmailValid = validateEmailUseCase(email)
+        if (!isEmailValid) {
+            emitViewState { viewState ->
+                viewState.copy(isAttendeeEmailError = true)
+            }
+            return
+        }
+
+        viewModelScope.launch {
+            val attendeeResult = createEventRepository.getAttendee(email)
+
+            val attendee = attendeeResult.getOrNull()
+
+            if (attendeeResult.isFailure || attendee == null) {
+                emitViewState { viewState ->
+                    viewState.copy(
+                        isAttendeeEmailError = true
+                    )
+                }
+                return@launch
+            }
+
+            emitViewState { viewState ->
+                viewState.copy(
+                    attendees = mutableListOf<Attendee>().also {
```

Comment:
How do you usually update items in the list? Adding data to the list, removing the data from the list, or maybe the most complex thing to update item property within the list.

Reply:
In your case I'd just do this:
```
attendees = listOf(attendee) + viewState.attendees
```
To update one item in the list, I use `map` like this:
```
attendees = attendees.map {
    if(it.id == attendeeToUpdateId) {
        it.copy(property = newValue)
    } else it
}
```
To remove one, I use `filter`:
```
attendees = attendees.filter { it.id != attendeeToRemoveId }
```